packages/yoga/src/AutoComplete/web/AutoComplete.jsx

```diff
@@ -200,33 +229,51 @@ const AutoComplete = React.forwardRef(
               second.toLowerCase().indexOf(inputValue)
                 ? -1
                 : 1,
-            )
-            .slice(0, 6);
+            );
 
-          const hasSuggestion = isOpen && Boolean(suggestionList.length);
+          if (!!inputValue && isOpen) {
+            setIsSuggestionsOpen(true);
+          }
 
           return (
             <Wrapper
               className={className}
               style={style}
               full={full}
-              isOpen={hasSuggestion}
+              isOpen={isSuggestionsOpen}
               {...getRootProps()}
               ref={ref}
+              onBlur={handleCloseSuggestions}
             >
               <Input
                 {...props}
+                error={error}
                 full={full}
                 onClean={cleanable => {
                   onClean(cleanable);
                   clearSelection();
+                  handleCloseSuggestions();
                 }}
                 {...getInputProps({
                   onFocus: () => (inputValue.length ? openMenu() : null),
                 })}
+                rightIcon={
+                  isSuggestionsOpen ? (
+                    <ChevronUp
+                      onClick={handleCloseSuggestions}
+                      aria-label="Close suggestions"
+                    />
+                  ) : (
+                    <ChevronDown
+                      onClick={handleOpenSuggestions}
+                      aria-label="Open suggestions"
```

Comment:
don't you think it is better to set `aria-label` as props? because gives the possibility of internationalization by the clients

Reply:
Hey, @diegomarcuz!
Great idea! Just pushed the changes for this implementation.

Reply:
This is a great point but thinking in all Yoga components, since the DS today doesn't give support to accessibility at all. 
We can talk more about it later, but these accessibility props could be in the System and we can pass it to all components, and also take a look at where we aren't handling it on the component itself.

Reply:
Great idea @frgiovanna. Mainly, because each component could have a specific behavior and requires some accessibility treatment.